README.md

```diff
@@ -21,60 +21,63 @@ predictable, and faster.
 
 ## Installation
 
-`prescient.el` is available on MELPA as four separate packages (one
+`prescient.el` is available on MELPA as six separate packages (one
 for the library, and the rest for integrating with other frameworks):
 
 * [`prescient`](https://melpa.org/#/prescient)
-* [`ivy-prescient`](https://melpa.org/#/ivy-prescient)
+* [`corfu-prescient`](https://melpa.org/#/corfu-prescient)
 * [`company-prescient`](https://melpa.org/#/company-prescient)
+* [`ivy-prescient`](https://melpa.org/#/ivy-prescient)
 * [`selectrum-prescient`](https://melpa.org/#/selectrum-prescient)
+* [`vertico-prescient`](https://melpa.org/#/vertico-prescient)
 
 The easiest way to install these packages is using
 [`straight.el`][straight.el]:
 
     (straight-use-package 'prescient)
-    (straight-use-package 'ivy-prescient)
+    (straight-use-package 'corfu-prescient)
     (straight-use-package 'company-prescient)
+    (straight-use-package 'ivy-prescient)
     (straight-use-package 'selectrum-prescient)
+    (straight-use-package 'vertico-prescient)
 
 However, you may install using any other package manager if you
 prefer.
 
 ## Usage
 
-* To cause [Ivy] to use `prescient.el` sorting and filtering, enable
-  `ivy-prescient-mode`.
+* To cause Emacs to use the `prescient` completion style for
+  filtering, enable `prescient-completion-mode`. To use `prescient`
+  only in the current buffer (such as for `completion-in-region`), see
+  `prescient-completion-local-mode`.
+
+* To cause [Corfu] to use `prescient.el` sorting and filtering, enable
+  `corfu-prescient-mode`, which uses `prescient-completion-local-mode`
+  in buffers where `corfu-mode` is enabled.
+
 * To cause [Company] to use `prescient.el` sorting, enable
   `company-prescient-mode`.
+
+* To cause [Ivy] to use `prescient.el` sorting and filtering, enable
+  `ivy-prescient-mode`.
+
 * To cause [Selectrum] to use `prescient.el` sorting and filtering,
   enable `selectrum-prescient-mode`.
+
+* To cause [Vertico] to use `prescient.el` sorting and filtering,
+  enable `vertico-prescient-mode`, which uses
+  `prescient-completion-local-mode` in the Vertico buffer.
```

Comment:
Maybe we should mention for Corfu and Vertico that the variables `completion-styles` etc. are modified? Also document a way for a user to prevent this, if they want to use their own settings?

Reply:
That is `corfu-prescient-enable-filtering` and `vertico-prescient-enable-filtering`, similar to `selectrum-prescient-enable-filtering`. I've listed those variables in their descriptions in the README.

Reply:
See https://github.com/radian-software/prescient.el/pull/131#discussion_r1008896900. I think it is better to be very explicit about the unusual modification of the `completion-styles` variable.